Source: Rasmus Thomsen <Rasmus.thomsen@protonmail.com>
Upstream: Issue reported, https://bugs.mysql.com/bug.php?id=77437
Reason: This is required for cmake to detect libressl since it
        doesn't have a OPENSSL_MAJOR_VERSION of 1, but 2

diff --git a/cmake/ssl.cmake b/cmake/ssl.cmake
index 19ac9a6..b5bf04d 100644
--- a/cmake/ssl.cmake
+++ b/cmake/ssl.cmake
@@ -176,7 +176,8 @@ MACRO (MYSQL_CHECK_SSL)
     IF(OPENSSL_INCLUDE_DIR AND
        OPENSSL_LIBRARY   AND
        CRYPTO_LIBRARY      AND
-       OPENSSL_MAJOR_VERSION STREQUAL "1"
+       OPENSSL_MAJOR_VERSION STREQUAL "1" OR
+       OPENSSL_MAJOR_VERSION STREQUAL "2"
       )
       SET(OPENSSL_FOUND TRUE)
     ELSE()
